tests/remotes/s3.py

```diff
@@ -99,7 +99,8 @@ def read_text(self, encoding=None, errors=None):
 
 
 @pytest.fixture
-def s3():
+def s3(test_config):
+    test_config.requires("s3")
```

Comment:
Might want to have --disable/enable for `s3` and `real_s3` separately in the future. But for now this will do.